portfolio/src/main/webapp/index.html

```diff
@@ -2,17 +2,38 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Jazmine's Portfolio</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
+      <h1>Jazmine's Portfolio</h1>
+      <p>Hello! My name is Jazmine Bullock.</p>
+      <img width = "340" src="images/pic-1.jpg" alt = "hey!"/>
+          <p>I am a Computer Science student at North Carolina A&T University.</p>
+          <p>I also am in the Blue and Gold marching machine! Go piccolos!</p>
+          <img width = "300" src="images/NCAT.png" alt = "school"/>
```

Comment:
Widths on img elements are better done in style sheets. Let me know if you need help with that. It's fine as-is for now.